src/clarity-angular/utils/focus-trap/index.ts

```diff
@@ -0,0 +1,11 @@
+/*
+* Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+* This software is released under MIT license.
+* The full license information can be found in LICENSE in the root directory of this project.
+*/
+ import { Type } from "@angular/core";
+ import {FocusTrapDirective} from "./focus-trap.directive";
+
+ export const FOCUS_TRAP_DIRECTIVES: Type<any>[] = [
+     FocusTrapDirective
+ ];
```

Comment:
It looks like we have one gemini test failing in CSS regression. I don't think that's a big deal. Maybe @hippee-lee can help get that resolved?